### DISCUSSION TITLE
zh/device-dev/kernel/kernel-small-apx-dll

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#715] [Announcements]
# zh/device-dev/kernel/kernel-small-apx-dll

基本概念 双向链表是指含有往前和往后两个方向的链表，即每个结点中除存放下一个节点指针外，还增加一个指向前一个节点的指针。其头指针head是唯一确定的。从双向链表中的任意一个结点开始，都可以很方便地访问它的前驱结点和后继结点，这种数据结构形式使得双向链表在查找时更加方便，特别是大量数据的遍历。由于双向链表具有对称性，能方便地完成各种插入、删除等操作，但需...

http://localhost:8080/zh/device-dev/kernel/kernel-small-apx-dll.html

**Comment by kuangyufei** [comment]
这个文档还不错
